lasagne/layers/conv.py

```diff
@@ -33,29 +33,27 @@ def conv_output_length(input_length, filter_size,
     stride : int
         The stride of the convolution operation.
 
-    border_mode : str, 'valid', 'full', 'same' or 'pad'
-        A string indicating the convolution border mode.
+    pad : int, tuple of int, 'full' or 'same' (default: 0)
```

Comment:
Can't be a tuple of int in this function, only a single value. (Also needs slight adaptation of the text below.)